mrtt-ui/src/components/Costs/CostsForm.js

```diff
@@ -131,7 +131,7 @@ const CostsForm = () => {
   } = useFieldArray({ name: 'percentageSplitOfActivities', control })
 
   const { siteId } = useParams()
-  const apiAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/registration_answers`
+  const interventionFormsUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/intervention_answers`
```

Comment:
this is going to screw up the `successCallback` and will be addressed in upcoming prs

Reply:
REthinking using different urls for this............ we should just stick to registration answers for all of them. Angelo suggested that was an option. I didng think it was a big deal until I got my hands dirty wit this.

Reply:
https://sparkgeo.slack.com/archives/C036D4TLVPT/p1663377679361169